components/insight/SRC/org/openmicroscopy/shoola/util/ui/colourpicker/RGBModel.java

```diff
@@ -76,50 +77,71 @@
 	/** The lookup table */
 	private String lut;
 	
+	/** The reverse intensity flag */
+	private boolean revInt;
+	
+	/** The original reverse intensity flag */
+	private boolean originalRevInt;
 	
     /**
      * Sets the value of the model to v without firing an event.
      * 
      * @param v value.
      */
-    private void setV(float v)
+    private void setV(float v, boolean reset)
```

Comment:
There's an Eclipse setting somewhere that can be turned on to make these Javadoc issues yellow warnings. (If you find it, show @pwalczysko!)